### PR TITLE
AST-204 - WC store notice tooltip to description.

### PR DESCRIPTION
**Pull request by GauravKhupse**
### Description
Updated tooltip to the description for store notice.

### Screenshots
https://share.bsf.io/X6ubbz7J

### Types of changes
WooCommerce description improvements.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/customizer/class-astra-customizer.php [hunk 1]
```diff
@@ -384,7 +384,11 @@ public function print_footer_scripts() {
 	                    			return \'&#\'+i.charCodeAt(0)+\';\';
 								});
 	                    	desc.remove();
-	                    	li_wrapper.append(" <i class=\'ast-control-tooltip dashicons dashicons-editor-help\'title=\'" + tooltip +"\'></i>");
+							if(ctrl["id"]){
```
[the remark this answers is not in the document]
Comment (GauravKhupse) [reply inc/customizer/class-astra-customizer.php]: Changes done!

File: inc/customizer/class-astra-customizer.php [hunk 2]
```diff
@@ -384,7 +384,12 @@ public function print_footer_scripts() {
 	                    			return \'&#\'+i.charCodeAt(0)+\';\';
 								});
 	                    	desc.remove();
-	                    	li_wrapper.append(" <i class=\'ast-control-tooltip dashicons dashicons-editor-help\'title=\'" + tooltip +"\'></i>");
+							console.log(ctrl["id"]);
```
Comment (imnavanath) [inline inc/customizer/class-astra-customizer.php:387]:
```suggestion
```

Comment (GauravKhupse) [reply inc/customizer/class-astra-customizer.php]: changes done!

File: inc/customizer/class-astra-customizer.php [hunk 3]
```diff
@@ -384,7 +384,11 @@ public function print_footer_scripts() {
 	                    			return \'&#\'+i.charCodeAt(0)+\';\';
 								});
 	                    	desc.remove();
-	                    	li_wrapper.append(" <i class=\'ast-control-tooltip dashicons dashicons-editor-help\'title=\'" + tooltip +"\'></i>");
+							if("woocommerce_demo_store_notice" === ctrl["id"]){
+								li_wrapper.append( tooltip);
+							} else {
+								li_wrapper.append(" <i class=\'ast-control-tooltip dashicons dashicons-editor-help\'title=\'" + tooltip +"\'></i>");
+							}
```
Comment (premanshup) [inline inc/customizer/class-astra-customizer.php:391]:
```suggestion
							if( "woocommerce_demo_store_notice" === ctrl["id"]){
								li_wrapper.append( tooltip );
							} else {
								li_wrapper.append( " <i class=\'ast-control-tooltip dashicons dashicons-editor-help\'title=\'" + tooltip +"\'></i>" );
							}	
```

Comment (GauravKhupse) [reply inc/customizer/class-astra-customizer.php]: Changes done!

**Pull request closed without merging**